scripts/Blood-Magic-Thaumcraft.zs

```diff
@@ -59,7 +59,7 @@ mods.thaumcraft.Infusion.addRecipe("LIFEINFUSER", <AWWayofTime:Altar>,
 [<minecraft:nether_star>, <gregtech:gt.metaitem.02:22977>, <gregtech:gt.metaitem.02:22977>, <AWWayofTime:imbuedSlate>, <TConstruct:heavyPlate:251>, <TConstruct:heavyPlate:501>, <TConstruct:heavyPlate:251>, <AWWayofTime:imbuedSlate>, <gregtech:gt.metaitem.02:22977>, <gregtech:gt.metaitem.02:22977>], 
 "victus 32, alienis 32, praecantatio 24, auram 16, fames 8, terra 8", <BloodArsenal:life_infuser>, 5);
 mods.thaumcraft.Research.addInfusionPage("LIFEINFUSER", <BloodArsenal:life_infuser>);
-mods.thaumcraft.Warp.addToResearch("LIFEINFUSER", 3);
+mods.thaumcraft.Warp.addToResearch("LIFEINFUSER", 50);
```

Comment:
Is this supposed to add 50 warp? Maybe you meant to add only 5

Reply:
It's supposed to be 50. It uses infinity and is completely broken. #2834

Reply:
Actually is it fixed now? At the bottom of #3953 it mentions GT items being blacklisted, but can you use it to mess with other stuff?

Reply:
... Please at least include a warning or some other way to tell it apart somehow.  Maybe something like a quest or custom icons

Reply:
I do think moving it to NE of the bound armor research might be appropriate, but I'm not 100% sure how to move them. And I'd probably want to add Teleposer research at the same time. Custom icon is no-go for me. Quest might be possible, but it would be UV, and wouldn't save you from getting the warp. I'm planning on doing quest rearranging/adding after this depending on the results.

Reply:
How about you restricted the research behind some other high level research that could optionally be required for the recipe. My main concern is I would be doing similar researches and nothing would make it apparent that this research is insanely warping. It would be easy to jump waay past 80 warp without realizing it which isn't a fun surprise.

Reply:
It's used to charge the life imbued armor as well as repair things (I thought it charged the bound, I've never used either). So putting it behind that would make sense. But I have to restart the client every time I test it, and that takes forever, so I was putting it off.